bitbots_localization/launch/localization.launch

```diff
@@ -1,16 +1,20 @@
 <launch>
     <!-- define parameters -->
-    <arg name="fieldname" default="feldraum" doc="Loads field settings, e.g. for the feldraum (f-031)." />
+    <arg name="fieldname" default="webots" doc="Loads field settings, e.g. for the feldraum (f-031)." />
```

Comment:
```suggestion
    <arg name="fieldname" default="feldraum" doc="Loads field settings (feldraum, webots, gazebo)." />
```
The default should be feldraum, we could automatically load webots if `sim:=true`...